extensions/spoons/init.lua

```diff
@@ -6,10 +6,37 @@ local module={}
 --- Spoons are Lua plugins for Hammerspoon.
 --- See http://www.hammerspoon.org/Spoons/ for more information
 
+--- hs.spoons.repos
+--- Variable
+--- Table containing the list of available Spoon repositories. The key
+--- of each entry is an identifier for the repository, and its value
+--- is a table with the following entries:
+---  * desc - Human-readable description for the repository
+---  * url - Base URL for the repository. For now the repository is assumed to be hosted in GitHub, and the URL should be the main base URL of the repository. Repository metadata needs to be stored under `docs/docs.json`, and the Spoon zip files need to be stored under `Spoons/`.
```

Comment:
@cmsj Do these assumptions make sense? Since we don't have a "download_url" metadata (nor I think we should - the download URL should not be hard-coded into the Spoon), it's getting built relative to the repo base URL, see L221.